apps/src/templates/instructions/CommitsAndReviewTab.jsx

```diff
@@ -0,0 +1,145 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import color from '@cdo/apps/util/color';
+import javalabMsg from '@cdo/javalab/locale';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
+import {ViewType} from '@cdo/apps/code-studio/viewAsRedux';
+import CodeReviewDataApiV2 from './codeReview/CodeReviewDataApiV2';
+import ReviewNavigator from './codeReview/ReviewNavigator';
+import Button from '@cdo/apps/templates/Button';
+
+export const VIEWING_CODE_REVIEW_URL_PARAM = 'viewingCodeReview';
+
+const CommitsAndReviewTab = ({
+  onLoadComplete,
+  channelId,
+  serverLevelId,
+  serverScriptId,
+  viewAsCodeReviewer,
+  viewAsTeacher,
+  codeReviewEnabled
+}) => {
+  const [loadingReviewData, setLoadingReviewData] = useState(false);
+
+  const dataApi = new CodeReviewDataApiV2(
+    channelId,
+    serverLevelId,
+    serverScriptId
+  );
+
+  const refresh = () => {
+    setLoadingReviewData(true);
+    // TODO: load review data
+    onLoadComplete();
+    setLoadingReviewData(false);
+  };
+
+  const loadPeers = async (onSuccess, onFailure) => {
+    try {
+      const response = await dataApi.getReviewablePeers();
+      onSuccess(response);
+    } catch (err) {
+      onFailure(err);
+    }
+  };
+
+  // channelId is not available on projects where the student has not edited the starter code.
+  // comments cannot be made on projects in this state.
+  if (!channelId) {
+    return (
+      <div style={{...styles.reviewsContainer, ...styles.messageText}}>
+        {javalabMsg.noCodeReviewUntilStudentEditsCode()}
+      </div>
+    );
+  }
```

Comment:
Can a code review be opened if the student has not edited the starter code?  (Wondering if we can eliminate this as a state that we need to handle here.)

Reply:
I think you're right, we'll want to hide the start review button instead of displaying this message. I'll handle that when I add the button